sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/models/package-info.java

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+/** Package containing classes for CommunicationIdentityClient. Azure Communication Identity Service. */
```

Comment:
I don't like the dangling phrase as the end.

Reply:
Hmmm I think this is an issue with autorest, this was autogenerated.